tests/wpt/web-platform-tests/workers/Worker_ErrorEvent_error.htm

```diff
@@ -4,6 +4,10 @@
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
 <script>
+// The worker events races with the window's load event; if the worker events
+// arrive first, the harness will detect the error event and fail the test.
+setup({ allow_uncaught_exception: true });
```

Comment:
Because it's what all the other tests do:
```
$ grep -ir allow_uncaught_exception tests/wpt/web-platform-tests/workers/
tests/wpt/web-platform-tests/workers/WorkerGlobalScope_ErrorEvent_filename.htm:setup({ allow_uncaught_exception: true });
tests/wpt/web-platform-tests/workers/WorkerGlobalScope_ErrorEvent_lineno.htm:setup({ allow_uncaught_exception: true });
tests/wpt/web-platform-tests/workers/Worker_ErrorEvent_type.htm:setup({ allow_uncaught_exception: true });
tests/wpt/web-platform-tests/workers/interfaces/WorkerGlobalScope/onerror/propagate-to-window-onerror.html:  allow_uncaught_exception: true,
tests/wpt/web-platform-tests/workers/WorkerGlobalScope_ErrorEvent_colno.htm:setup({ allow_uncaught_exception: true });
tests/wpt/web-platform-tests/workers/constructors/SharedWorker/same-origin.html:setup({allow_uncaught_exception: true});
tests/wpt/web-platform-tests/workers/constructors/Worker/AbstractWorker.onerror.html:setup({allow_uncaught_exception:true});
tests/wpt/web-platform-tests/workers/constructors/Worker/same-origin.html:setup({allow_uncaught_exception: true});
tests/wpt/web-platform-tests/workers/Worker_ErrorEvent_message.htm:setup({ allow_uncaught_exception: true });
tests/wpt/web-platform-tests/workers/Worker_ErrorEvent_bubbles_cancelable.htm:setup({ allow_uncaught_exception: true });
tests/wpt/web-platform-tests/workers/WorkerGlobalScope_ErrorEvent_message.htm:setup({ allow_uncaught_exception: true });
tests/wpt/web-platform-tests/workers/Worker_ErrorEvent_filename.htm:setup({ allow_uncaught_exception: true });
tests/wpt/web-platform-tests/workers/Worker_ErrorEvent_lineno.htm:setup({ allow_uncaught_exception: true });
tests/wpt/web-platform-tests/workers/semantics/reporting-errors/004.html:setup({allow_uncaught_exception:true});
tests/wpt/web-platform-tests/workers/semantics/reporting-errors/002.html:setup({allow_uncaught_exception:true});
tests/wpt/web-platform-tests/workers/semantics/reporting-errors/001.html:setup({allow_uncaught_exception:true});
tests/wpt/web-platform-tests/workers/semantics/reporting-errors/003.html:setup({allow_uncaught_exception:true});
tests/wpt/web-platform-tests/workers/semantics/run-a-worker/003.html:setup({allow_uncaught_exception: true});
```
The odd one out is `data-url.html`, since some of the tests should raise an exception and some should not, so in that case I explicitly blocked the exception from propagating.